### ISSUE TITLE
Better navigation for public folder

### ISSUE THREAD

**Issue by WanyiChen** [solid/solid-ui#57]
Hi, I'm relatively new to GitHub, so I'm not sure if this is the right place for this issue. I found it quite hard to navigate through my Solid Pod's public folder. For example, when I get to here:
![screen shot 2018-12-13 at 5 55 32 pm](https://user-images.githubusercontent.com/15915356/49972567-160f8980-ff00-11e8-9024-fa3f80196055.png)
If I click on the x, the page becomes completely blank, and there's no way to restore it. I think it's good to either have a way to restore the page, or get rid of the x, since the x does not really make sense here. Also, if I finish editing my public folder, there's no direct way to navigate back to this page to edit other folders:
![screen shot 2018-12-13 at 5 57 30 pm](https://user-images.githubusercontent.com/15915356/49972705-7f8f9800-ff00-11e8-9f8f-37ebf8fd7bf1.png)
I think Solid is a great project, but it can potentially attract more people if it had a better UI!

**Comment by scenaristeur** [comment]
+ there are lots of mistakes,  when i want to see détails name of file change to description
![screenshot_20181214-004725_chrome](https://user-images.githubusercontent.com/4020744/49974555-3eb47500-ff3a-11e8-8b2b-47286f7ed8bf.jpg)
![screenshot_20181214-004735_chrome](https://user-images.githubusercontent.com/4020744/49974590-5e4b9d80-ff3a-11e8-8e0d-3f0175ff650f.jpg)
My POD is on solid.community provider

**Comment by jasonmerry** [comment]
I'd love to take this on as and when I can. Already forked...

1st thing I would do, remove all the images being used for icons and replace them with a custom bespoke webfont made up from fontello.com into a lot less files. 

2nd, define the style of the tool in both colour and layout so everything is consistent. 
The favicon will be a great place to start.

**Comment by SharonStrats** [comment]
I don't believe we have this view anymore.  I have clicked around navigation and I just see Your Storage which doesn't have the x next to it, it only has the link button. I actually don't see a way to delete files at all.
